app/build.gradle.kts

```diff
@@ -34,6 +69,7 @@ android {
 			// Use different application id to run release and debug at the same time
 			applicationIdSuffix = ".debug"
 			resValue("string", "app_name", "@string/app_name_debug")
+			versionNameSuffix =  "-debug-" + getBranchName() + "-" + getGitHash() + (if (isTreeClean()) "-treeClean" else "-treeDirty")
```

Comment:
I think I would prefer it only append `-dirty` if the tree is not clean and nothing when it is.

Reply:
Okay, will change that